packages/flame/lib/src/effects/opacity_effect.dart

```diff
@@ -68,14 +68,8 @@ class OpacityEffect extends ComponentEffect<HasPaint> {
     final currentAlpha = target.getAlpha(paintId: paintId);
     final deltaAlpha =
         (_alphaOffset * deltaProgress) + _roundingError * deltaProgress.sign;
-    final remainder = deltaAlpha.remainder(1.0).abs();
-    _roundingError = remainder >= 0.5 ? -1 * (1.0 - remainder) : remainder;
     var nextAlpha = (currentAlpha + deltaAlpha).round();
-    if (nextAlpha < 0) {
-      _roundingError += nextAlpha.abs();
-    } else if (nextAlpha > 255) {
-      _roundingError += nextAlpha - 255;
-    }
```

Comment:
I have a feeling I wrote this code, I wonder why I made it so damn complicated. :sweat_smile:

Reply:
It does show your name in GitLens 😄

![image](https://user-images.githubusercontent.com/33748002/191580801-ee63f889-f78a-4703-b5f4-5ed35edf1faa.png)